sympy/stats/stochastic_process_types.py

```diff
@@ -225,7 +241,8 @@ def __call__(self, time):
         if time not in self.index_set:
             raise IndexError("%s is not in the index set of %s"%(time, self.symbol))
         func_obj = Function(self.symbol)(time)
-        pspace_obj = StochasticPSpace(self.symbol, self)
+        distribution = getattr(self, 'distribution', None)
```

Comment:
```suggestion
        distribution = self.distribution
```

Reply:
This is because all processes don't necessarily have `distribution`, so used `getattr`, for instance, say, CTMC